tools/beman-tidy/beman_tidy/lib/checks/beman_standard/license.py

```diff
@@ -70,7 +70,33 @@ def fix(self):
         )
 
 
-# TODO LICENSE.APACHE_LLVM
+@register_beman_standard_check("LICENSE.APACHE_LLVM")
+class LicenseApacheLLVMCheck(LicenseBaseCheck):
+    def __init__(self, repo_info, beman_standard_check_config):
+        super().__init__(repo_info, beman_standard_check_config)
+
+    def check(self):
+        # path/to/repo/LICENSE content
+        content = self.read()
+
+        # infra LICENSE content
+        with open("../../LICENSE", "r") as f:
+            infra_license = f.read()
+
+        if content != infra_license:
```

Comment:
```suggestion
from pathlib import Path
...
        # Compare LICENSE file stored at self.path with the reference one.
        target_license = self.path
        ref_license = Path(__file__).resolve().parent.parent.parent / "LICENSE"
        if not filecmp.cmp(target_license,  ref_license, shallow=False):
```

../../LICENSE should not work if we run the script from other path. We should use Path.parent - check https://github.com/bemanproject/infra/blob/main/tools/beman-tidy/beman_tidy/lib/utils/git.py#L71

also, I think it's better to use filecmp.cmp